### DISCUSSION TITLE
get times and kstpkper from input flopy.mf6.modflowtdis, before model execution

### DISCUSSION THREAD

**Discussion by chenmingzhang** [modflowpy/flopy#1371] [Ideas]
A modeling project I am working on needs to use field time series data to construct time dependent boundary conditions (CHD, WEL, RCH), before model execution. This will require interpolation against the field  time series data, using **a time array formed by the input for temporal discretization package** (e.g.,  ntsp perlen and tmulti) ,  which are only availalbe after the model is executed. (e.g., headobj.get_times() )

a example would be as below:

```
input_recharge_rate = interpolation ( field_data_time_array, field_data_recharge_array, model_time_array) 
```
Just wondering if i can get model_time_array from tdis, after running 

```
tdis = flopy.mf6.ModflowTdis(sim, 
                             time_units='DAYS',
                             nper=nper, 
                             perioddata=dis_perioddata_ay
                            )
```

maybe tdis.get_times or tdis.get_kstpkper  ? I read the manual and method but did not find this. 

I know that for spatial discretisation, the grid information can be assessed from **gwf.modelgrid**

Thanks!

**Comment by christianlangevin** [comment]
try:

```
gwf.modeltime.totim
```

I haven't used this, but I see it in the flopy code.

**Reply by chenmingzhang** [reply]
Thanks, Christian, it works. Cheers, chenming
